obspy/io/ah/core.py

```diff
@@ -257,7 +257,10 @@ def _pack_trace_with_ah_dict(trace, packer):
         packer.pack_int(CODESIZE)
         packer.pack_fstring(CODESIZE, tr.stats.ah.station.code)
         packer.pack_int(CHANSIZE)
-        packer.pack_fstring(CHANSIZE, tr.stats.ah.station.channel)
+        try:
+            packer.pack_fstring(CHANSIZE, tr.stats.channel)
+        except:
+            packer.pack_fstring(CHANSIZE, tr.stats.ah.station.channel)
```

Comment:
The AH files usually store some event information as well, therefore I wrote two functions. If the AH file was read all those meta-informations go into `trace.stats.ah`. Was not sure how to handle it in a proper way